src/tools/kata-ctl/src/arch/x86_64/mod.rs

```diff
@@ -43,9 +58,8 @@ mod arch_specific {
         Ok(())
     }
 
-    pub fn check() -> Result<()> {
-        println!("INFO: check: x86_64");
-
+    pub fn check(args: &str) -> Result<()> {
```

Comment:
- I think we can drop all the arch-specific `check()` functions now as they are replaced by `get_checks()`.
- I'm not sure we need `args` currently so we can either remove it and add when we need it, or keep it. But if we keep it, you'll need to change `check_cpu()` to be `check_cpu(args: &str)`.

Reply:
Yes I share this decision